tests/data/test_websockets.py

```diff
@@ -1,33 +1,33 @@
+from datetime import datetime
+import pandas as pd
 import pytest
+from msgpack.ext import Timestamp
 
 from alpaca.common.websocket import BaseStream
 from alpaca.data.enums import Exchange
 from alpaca.data.models import Bar, Trade
 
 
 @pytest.fixture
-def ws_client():
-    client = BaseStream("endpoint", "key-id", "secret-key")
-    return client
+def ws_client() -> BaseStream:
+    """Socket client fixture with pydantic models as output."""
+    return BaseStream("endpoint", "key-id", "secret-key")
 
 
 @pytest.fixture
-def raw_ws_client():
-    raw_client = BaseStream("endpoint", "key-id", "secret-key", raw_data=True)
-    return raw_client
+def raw_ws_client() -> BaseStream:
+    """Socket client fixture with raw data output."""
+    return BaseStream("endpoint", "key-id", "secret-key", raw_data=True)
 
 
 @pytest.fixture
-def timestamp():
-    class MockTimestamp:
-        def __init__(self, _seconds, _nanoseconds):
-            self.seconds = _seconds
-            self.nanoseconds = _nanoseconds
+def timestamp() -> datetime:
+    """Msgpack mock timestamp."""
+    return datetime(year=2023, month=1, day=10, hour=10, second=10)
 
-    return MockTimestamp(0, 0)
 
-
-def test_cast(ws_client, raw_ws_client, timestamp):
+def test_cast(ws_client: BaseStream, raw_ws_client: BaseStream, timestamp: Timestamp):
```

Comment:
```suggestion
def test_cast(ws_client: BaseStream, raw_ws_client: BaseStream, timestamp: datetime):
```

Reply:
if we're not making that breaking change we should have msgpack.ext.Timestamp in the test fixture imo